### PR TITLE
Created What to Expect Page (Attempt 2)

### PR DESCRIPTION
**Pull request by andrewwoods**
Created new page - What to Expect - and uses route /what-to-expect.

### CODE REVIEW

**Comment by jeremeamia** [comment]
What the merge?

File: resources/views/what-to-expect.blade.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                    <h1>What to Expect</h1>
+                </div>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-8 col-md-offset-2">
+                <h2>Environment</h2>
+                <p>Pacific NorthWest PHP (PNWPHP) Conference maintains a friendly, 
+                professional, and welcoming environment. Developers of all skill 
+                levels are welcome. We have a Code of Conduct that all attendees
```
Comment (jeremeamia) [inline resources/views/what-to-expect.blade.php:18]: link to CoC

File: resources/views/what-to-expect.blade.php [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                </div>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-8 col-md-offset-2">
+                <h2>Environment</h2>
+                <p>Pacific NorthWest PHP (PNWPHP) Conference maintains a friendly, 
+                professional, and welcoming environment. Developers of all skill 
+                levels are welcome. We have a Code of Conduct that all attendees 
+                must agree to when purchasing their ticket. We take all Code of
```
Comment (jeremeamia) [inline resources/views/what-to-expect.blade.php:19]: Not true, they are not forced to agree with it when purchasing their ticket.

File: resources/views/what-to-expect.blade.php [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        </div>
+        <div class="row">
+            <div class="col-md-8 col-md-offset-2">
+                <h2>Environment</h2>
+                <p>Pacific NorthWest PHP (PNWPHP) Conference maintains a friendly, 
+                professional, and welcoming environment. Developers of all skill 
+                levels are welcome. We have a Code of Conduct that all attendees 
+                must agree to when purchasing their ticket. We take all Code of 
+                Conduct reports seriously, and our Code of Conduct will be 
+                enforced. For more information, please see the Madison PHP
```
Comment (jeremeamia) [inline resources/views/what-to-expect.blade.php:21]: Madison PHP, eh?

File: resources/views/what-to-expect.blade.php [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+                Conduct reports seriously, and our Code of Conduct will be 
+                enforced. For more information, please see the Madison PHP 
+                Conference Code of Conduct.</p>
+
+                <h2>Getting to PNWPHP</h2>
+                <p>Parking can be difficult in Downtown Seattle. We suggest taking 
+                public transportation if you live nearby. Several bus routes, and 
+                Link Light Rail have stops nearby. If you need to drive, try to 
+                carpool if possible. This will cut down on traffic, and parking 
+                issues. Directions to the conference are located on our Venue
```
Comment (jeremeamia) [inline resources/views/what-to-expect.blade.php:29]: What Venue page?

File: resources/views/what-to-expect.blade.php [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+                levels are welcome. We have a Code of Conduct that all attendees 
+                must agree to when purchasing their ticket. We take all Code of 
+                Conduct reports seriously, and our Code of Conduct will be 
+                enforced. For more information, please see the Madison PHP 
+                Conference Code of Conduct.</p>
+
+                <h2>Getting to PNWPHP</h2>
+                <p>Parking can be difficult in Downtown Seattle. We suggest taking 
+                public transportation if you live nearby. Several bus routes, and 
+                Link Light Rail have stops nearby. If you need to drive, try to
```
Comment (jeremeamia) [inline resources/views/what-to-expect.blade.php:27]: link to light rail?

File: resources/views/what-to-expect.blade.php [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+                public transportation if you live nearby. Several bus routes, and 
+                Link Light Rail have stops nearby. If you need to drive, try to 
+                carpool if possible. This will cut down on traffic, and parking 
+                issues. Directions to the conference are located on our Venue 
+                page.</p>
+
+                <h2>Arriving</h2>
+                <p>When you arrive at the Impact Hub, the Main Entrance on 2nd Ave 
+                S, just south of S Washington St. When you walk in, you will see a 
+                registration table, which is where you check in .</p>
```
Comment (jeremeamia) [inline resources/views/what-to-expect.blade.php:35]: ...registration table _on your right_, which is where...

File: resources/views/what-to-expect.blade.php [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+                <h2>Talks</h2>
+                <p>The main event room is to the right of the registration area.
```
Comment (jeremeamia) [inline resources/views/what-to-expect.blade.php:38]: The main event room is **on the first floor, to the left of the elevator**.

File: resources/views/what-to-expect.blade.php [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+                you have already purchased your ticket, please send us an email to 
+                let us know about your dietary restrictions. We will try to work 
+                with your dietary restrictions. We will do our best to accommodate 
+                you, but we may not be able to.</p>
+
+                <h2>Getting Help</h2>
+                <p>If you need assistance at any time, any one of our organizers 
+                would be happy to help you out. The organizers will be moving 
+                around the venue, but the easiest place to find them will be at the 
+                registration table (with the red Madison PHP logo on the front of
```
Comment (jeremeamia) [inline resources/views/what-to-expect.blade.php:67]: Madison PHP? :stuck_out_tongue:

**Comment by andrewwoods** [comment]
I pulled in all the main repo updates, and got messages like "this thing is out of sync with such-and-such". I just kept merging until they went away. I'm gonna see if I can clean it up.

**Comment by jeremeamia** [comment]
Weird stuff happened in the rollback a couple weeks ago. You could try applying your changes to a fresh checkout if all else fails.

**Comment by jeremeamia** [comment]
Closing in favor of #47

**Pull request closed without merging**